src/nspawn/nspawn-mount.c

```diff
@@ -275,20 +288,22 @@ int mount_all(const char *dest,
                 unsigned long flags;
                 bool fatal;
                 bool userns;
+                bool netns;
```

Comment:
I suggest to rename:
* `userns` -> `in_inner_child`
* `netns` -> `private_net_only`


Maybe a bit clearer?